universe/jupyterlab/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
+  "scm": "https://github.com/portworx/dcos-jupyterlab-service/releases/tag/px_1.3-1.3.0-0.35.4",
+  "maintainer": "support@portworx.com",
+  "website": "http://jupyter.org",
+  "description": "The JupyterLab Notebook is an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. Uses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. Installation Documentation: https://github.com/dcos/examples/tree/master/jupyterlab",
```

Comment:
Use all the fields from this version of jupyter service. Just use portworx related modifications from the older branch.